src/components/generic/movieModal/index.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react'
+import {FC} from 'react'
+
+import ModalContent from './modalContent'
+
+interface Data {
+  actors: string;
+  description: string;
+  directors: string;
+  genres: string[];
+  id: string;
+  imdb: string;
+  poster_big: string;
+  rating: number;
+  title: string;
+  trailer: string;
+  year: number;
+  items:[]
+}
+
+export interface Props {
+  open:boolean;
+  handleOpen(imdb:string):void;
+  handleClose():void;
+  data:Data;
+}
+
+ const MovieModal:FC<Props> = (props) => {
```

Comment:
Why was this component created? One can directly use `ModalContent` instead of using this component `MovieModal`. What's the reasoning behind it?

Reply:
This was intended to be as a container component. I was planning to add more stuff to it and manages them all. like video preview etc..

Reply:
Alright, understood. Generally doing this from the beginning is not a good idea. Don't think for the future if you are not gonna implement it.